### PR TITLE
6.4 release notes

### LINKED ISSUE #10853: 6.4: Create release notes
The place holder for the usual release notes pr.

### PR DESCRIPTION
**Pull request by pdurbin**
**What this PR does / why we need it**:

We always write release notes! 😄 

**Which issue(s) this PR closes**:

- Closes #10853

Preview at https://github.com/IQSS/dataverse/blob/10853-6.4-release-notes/doc/release-notes/6.4-release-notes.md

### CODE REVIEW

**Review by ofahimIQSS — COMMENTED** [review]
test review

File: doc/release-notes/6.4-release-notes.md [hunk 1]
```diff
@@ -0,0 +1,499 @@
...
+
+```shell
+http://localhost:8080/api/admin/clearThumbnailFailureFlag/$FILE_ID
+```
+
+12\. PermaLinks with custom base-url
+
+If you currently use PermaLinks with a custom `base-url`: You must manually append `/citation?persistentId=` to the base URL to maintain functionality.
+
+If you use a PermaLinks without a configured `base-url`, no changes are required.
```
Comment (ofahimIQSS) [inline doc/release-notes/6.4-release-notes.md:499]: Reviewed and Proof-read doc/release-notes/6.4-release-notes.md

**Review by ofahimIQSS — APPROVED** [review]
Verified the documentation - no issues found on my end

File: doc/release-notes/6.4-release-notes.md [hunk 2]
```diff
@@ -310,7 +310,7 @@ The `/api/dataverses/{identifier}/metadatablocks` endpoint has been fixed to not
 
 ### Feature Flags Can Be Listed
 
-It now possible to list all feature flags and see if they are enabled or not. See #10732 and [the guides](https://guides.dataverse.org/en/6.4/api/native-api.html#list-all-feature-flags).
+It now possible to list all feature flags and see if they are enabled or not. See also [the guides](https://guides.dataverse.org/en/6.4/api/native-api.html#list-all-feature-flags) and #10732.
```
Comment (ofahimIQSS) [inline doc/release-notes/6.4-release-notes.md:313]: @pdurbin - please add "is" after It in the beginning

**Review by ofahimIQSS — APPROVED** [review]
Reviewed change - looks good now.

**Review by ofahimIQSS — APPROVED** [review]

**Review by sekmiller — APPROVED** [review]
No objections. I caught/fixed one typo on the current version number

**Review by ofahimIQSS — APPROVED** [review]
reviewed and approved

**Review by ofahimIQSS — APPROVED** [review]
reviewed and approved

**Review by ofahimIQSS — APPROVED** [review]
reviewed and approved

**Review by ofahimIQSS — APPROVED** [review]
reviewed and approved

**Review by ofahimIQSS — APPROVED** [review]
reviewed and approved

**Review by ofahimIQSS — APPROVED** [review]
reviewed and approved

**Review by ofahimIQSS — APPROVED** [review]
reviewed and approved

**Review by ofahimIQSS — APPROVED** [review]
reviewed and approved

**Review by ofahimIQSS — APPROVED** [review]
Reviewed and Approved

**Review by ofahimIQSS — APPROVED** [review]
Reviewed and Approved

**Pull request merged**